webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.tsx

```diff
@@ -29,10 +29,12 @@ function CoreMenuOptions({handleOnSelect}: Props) {
         />
     );
 
+    const tomorrowClickHandler = useCallback((e) => handleOnSelect(e, tomorrow9amTime.valueOf()), [handleOnSelect, tomorrow9amTime]);
```

Comment:
`tomorrow9amTime` and `nextMonday` on line 52 are, if I am not mistaken, objects, so a new reference on every render. So the dependency should look into `tomorrow9amTime.valueOf()`. If not, the callback won't memoize anything (will regenerate on every render).

Another way to deal with it, is just to store the time instead of the moment object. Something like...

```
const tomorrow9amTime = moment().
    tz(userTimezone).
    add(1, 'days').
    set({hour: 9, minute: 0, second: 0, millisecond: 0}).
    valueOf();
```

That way you just have to deal with a `number` instead of an object.